src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    margin: 0 auto 40px;
+    height: 134px;
+  
+    width: 160px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+  
+    font-size: $font-size-m;
+    font-weight: $font-weight-m;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  
+    font-size: $font-size-s;
+    font-weight: $font-weight-s;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+  
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews {
+    font-size: $font-size-s;
+    font-weight: $font-weight-s;
+    line-height: 14px;
+    color: $main-text-color;
+  }
+
+  &__price {
+    &-section {
+      margin-bottom: 16px;
+
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    &-text {
+      font-size: $font-size-m;
+      font-weight: $font-weight-s;
+      line-height: 18px;
+      color: $secondary-text-color;
+    }
+
+    &-value {
+      font-size: $font-size-l;
+      font-weight: $font-weight-l;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
```

Comment:
Don't do it, because it can make a mess. better make separate selectors without nesting for each element
`&__price-section` and so on